content/docs/reference-react.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: react-api
-title: React Top-Level API
+title: React Üst Düzey API
```

Comment:
Hi @alioguzhan, i was thinking about page name so i'm gonna ask a question to you, should we consider the **Top-Level** word as `Üst Düzey` as now, or we just evaluate it much semantically like `Temel` ?

Reply:
Hey, since `Top-Level` means `Highest level` just `Ust Duzey` may not meet the meaning of `Top-Level`. I believe `En Üst Düzey API` fits.

Reply:
Ok. I am not that much sure about this now. I believe that the correct approach is still `En Ust Duzey API` but it sounds to complex. Maybe we can get rid of `En` and just go with `Ust Duzey API`. What do you think guys @suyalcinkaya @ozcanzaferayan @helloanil

Reply:
Alright, for now i'm waiting for thoughts from others. By the way, if we want to make understandable it for users, i think we should do **localization** not **_translation_**.

On this case `Ust Duzey` meaning is more abstract than `Temel`. Because using Top-Level word is creating mental context on users mind and makes it more complicated.

**For example:**

`Ust Duzey` _case_: 
createElement() React'in üst düzey API'larından biridir.

`Temel` _case_: 
createElement() React'in temel API'larından biridir.

Reply:
For the present, we assume it's fine, we'll change it later if necessary.